lib/validateMessage.js

```diff
@@ -153,16 +153,18 @@ function validateScope(isValid, scope) {
       error('a scope is required !');
       isValid = false;
     }
-    if (isValid && multipleScopesAllowed) {
-      scopes.forEach(validateIndividualScope);
-    }
-    if (isValid && !multipleScopesAllowed) {
-      if (scopes.length > 1) {
-        error('only one scope can be provided !');
-        isValid = false;
+    if (scopes.length > 0) {
```

Comment:
Maybe to avoid more `if` nesting at this point, we should inverse the check and return `isValid`? For example

```js
if (!scopes) {
  return isValid;
}
```